lib/src/values/logic_value.dart

```diff
@@ -476,7 +476,8 @@ abstract class LogicValue {
   /// LogicValue.ofString('0101').getRange(0, 7);   // Error - bad inputs end > length-1
   /// ```
   ///
-  LogicValue getRange(int startIndex, int endIndex) {
```

Comment:
Make sure to update the doc comments about behavior if `endIndex` is not provided